baml_language/sdk_tests/crates/csharp/phase6_slice/verify_compile_negative.sh

```
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly script_dir="$({ cd -- "$(dirname -- "${BASH_SOURCE[0]}")"; pwd; })"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)